apps/explorer/src/pages/validator/ValidatorStats.tsx

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useMemo } from 'react';
+
+import { DelegationAmount } from './DelegationAmount';
+
+import type { Validator } from '~/pages/validator/ValidatorDataTypes';
+
+import { Card } from '~/ui/Card';
+import { Heading } from '~/ui/Heading';
+import { Stats } from '~/ui/Stats';
+import { getStakedPercent } from '~/utils/getStakedPercent';
+
+type StatsCardProps = {
+    validatorData: Validator;
+    totalValidatorStake: bigint;
+    epoch: number;
+};
+
+export function ValidatorStats({
+    validatorData,
+    epoch,
+    totalValidatorStake,
+}: StatsCardProps) {
+    // TODO: add missing fields
+    // const numberOfDelegators = 0;
+    //  const selfStake = 0;
+    //  const lastEpoch = 0;
+    //  const totalRewards =  0;
+    //  const networkStakingParticipation = 0;
+    //  const votedLastRound =  0;
+    //  const tallyingScore =  0;
+    //  const lastNarwhalRound = 0;
+
+    const validator = useMemo(() => {
+        const { sui_balance, starting_epoch, delegation_token_supply } =
+            validatorData.fields.delegation_staking_pool.fields;
+
+        const num_epochs_participated = epoch - starting_epoch;
+
+        const APY = Math.pow(
+            1 +
+                (sui_balance - delegation_token_supply.fields.value) /
+                    delegation_token_supply.fields.value,
+            365 / num_epochs_participated - 1
+        );
+
+        return {
+            apy: APY ? APY : 0,
+            delegatedStakePercentage: getStakedPercent(
+                validatorData.fields.stake_amount,
+                totalValidatorStake
+            ),
+            totalStake: validatorData.fields.stake_amount,
+        };
+    }, [validatorData, epoch, totalValidatorStake]);
+
+    return (
+        <div className="flex w-full flex-col gap-5 md:mt-8 md:flex-row">
+            <div className="max-w-[480px] basis-full md:basis-2/5">
```

Comment:
same here, why the px max width? Seems like this could easily just be done with variable flex amounts or grid?